feature/experimental/qos/ate_tests/mixed_sp_wrr_traffic_test/mixed_sp_wrr_traffic_test.go

```diff
@@ -392,36 +396,36 @@ func TestMixedSPWrrTraffic(t *testing.T) {
 		},
 		"intf1-af3": {
 			frameSize:             1000,
-			trafficRate:           40,
+			trafficRate:           32,
```

Comment:
Could you please provide the reason for trafficRate change?

Reply:
@saarvind6481 Can you address this comment?

Reply:
Also, should the [README.md](https://github.com/openconfig/featureprofiles/tree/c7c2f18cb125f119ea57782caa06a0aa2d3090e1/feature/experimental/qos/ate_tests/mixed_sp_wrr_traffic_test/README.md) be updated aswell?